src/Comparator.php

```diff
@@ -16,7 +16,7 @@
 class Comparator
 {
     /**
-     * Whether $version1 > $version2.
+     * Evaluate the expression: $version1 > $version2.
```

Comment:
``Evaluates`` would be better. It would mean that prefixing it with ``this method `` (or ``Comparator::greaterThan `` would produce a valid sentence, making things look better when you generate an API doc